source/language_service/src/wrapper_refactor/tests.rs

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::{code_action, test_utils::compile_project_with_markers_no_cursor};
+use expect_test::expect;
+use qsc::line_column::{Encoding, Position, Range};
+
+fn get_wrapper_text(source: &str, op_name: &str) -> String {
+    let (compilation, _targets) =
+        compile_project_with_markers_no_cursor(&[("<source>", source)], false);
```

Comment:
You can just use the simple `compile_with_markers` helper here

Reply:
On second thought, this doesn't make sense. The tests don't use cursors (they don't use any markers), so using compile_with_markers gives an error. It works only with compile_project_with_markers_no_cursor.

Reply:
I think using a marker would have worked too instead of hardcoding a position (then you would have also been testing the refactors appear in the location you meant to). 

I prefer uniformity but it's not that important